plugins/rikki/heroeslounge/classes/mmr/MMRFetcher.php

```diff
@@ -21,8 +25,12 @@ public static function updateMMRs()
         $sloths = SlothModel::all();
 
         foreach ($sloths as $sloth) {
-            set_time_limit(30);
-            MMRFetcher::updateMMRHeroesProfile($sloth);
+            set_time_limit(60);
+            $throttleTime = MMRFetcher::updateMMRHeroesProfile($sloth);
+
+            if ($throttleTime > 0) {
+                sleep($throttleTime);
+            }
```

Comment:
We will have to see if this works on our server. I had bad experiences with increasing time limit in the past